src/xdr/Stellar-transaction.x

```diff
@@ -873,7 +936,53 @@ enum TransactionResultCode
     txNO_ACCOUNT = -8,           // source account not found
     txINSUFFICIENT_FEE = -9,     // fee is too small
     txBAD_AUTH_EXTRA = -10,      // unused signatures attached to transaction
-    txINTERNAL_ERROR = -11       // an unknown error occured
+    txINTERNAL_ERROR = -11,      // an unknown error occured
+
+    txNOT_SUPPORTED = -12,        // transaction type not supported
+    txFEE_BUMP_INNER_FAILED = -13 // fee bump inner transaction failed
+};
+
+// InnerTransactionResult must be binary compatible with TransactionResult
+// because it is be used to represent the result of a Transaction.
+struct InnerTransactionResult
+{
+    int64 feeCharged;
```

Comment:
is there ever a case where `feeCharged` will not be zero? It seems that the test cases in src/transactions/test/FeeBumpTransactionTests.cpp assert that the `feeCharged` in the inner transaction result is zero. If that's the case then it would be helpful to add a comment indicating that `feeCharged` should be ignored and it's only there so that `InnerTransactionResult` is binary compatible with `TransactionResult`.

Reply:
Done.